src/Public/LogRhythm/Admin/Get-LRListGuidByName.ps1

```diff
@@ -27,34 +27,23 @@ Function Get-LRListGuidByName {
 
     [CmdletBinding()]
     Param(
-        [Parameter(Mandatory=$true,Position=0)]
+        [Parameter(Mandatory = $false, Position = 0)]
```

Comment:
Good catch, the LrList functions were the first LogRhythm cmdlets I added to the module, and I didn't have my style quite nailed down at the time.